Modules/_tracemalloc.c

```diff
@@ -1259,6 +1271,13 @@ trace_to_pyobject(unsigned int domain, trace_t *trace,
     }
     PyTuple_SET_ITEM(trace_obj, 2, obj);
 
+    obj = PyLong_FromLong(trace->traceback->total_nframe);
```

Comment:
Since it's unsigned, I suggest to use PyLong_FromUnsignedLong() to avoid the risk of introducing a compiler warning.

Reply:
Good catch. Done.